bulk-ingester/src/integration-test/java/org/alfresco/hxi_connector/bulk_ingester/processor/BulkIngestionProcessorIntegrationTest.java

```diff
@@ -77,29 +79,36 @@ void cleanUp()
     @Test
     void shouldPublishAllNodesFromDb()
     {
-        // given
```

Comment:
Generally we have these comments in larger tests so we can tell how it's structured.